hack/make/.build-deb/rules

```diff
@@ -6,7 +6,7 @@ override_dh_gencontrol:
 	# if we're on Ubuntu, we need to Recommends: apparmor
 	echo 'apparmor:Recommends=$(shell dpkg-vendor --is Ubuntu && echo apparmor)' >> debian/docker-engine.substvars
 	# if we are building experimental we reccomend yubico-piv-tool
-	echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~)")' >> debian/docker-engine.substvars
+	echo 'yubico:Recommends=$(shell [ "$DOCKER_EXPERIMENTAL" ] && echo "yubico-piv-tool (>= 1.1.0~), libykcs11-1 (>= 1.1.0~)")' >> debian/docker-engine.substvars
```

Comment:
I see https://packages.debian.org/sid/yubico-piv-tool, but https://packages.debian.org/sid/libykcs11-1 is 404 -- where does this other package come from? :confused: